### PR TITLE
Registry

### PR DESCRIPTION
**Pull request by Zapadlo**

### CODE REVIEW

File: service-files/registry-sidekick.service [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+[Service]
+ExecStart=/bin/sh -c "\
+  etcdctl set /skydns/local/skydns/registry '{"host":"127.0.0.1"}'; \
+  etcdctl set /vulcand/backends/repo-mirror/backend '{\"Type\": \"http\"}'; \
+  etcdctl set /vulcand/frontends/repo-mirror/frontend '{\"Type\": \"http\", \"BackendId\": \"docker-registry\", \"Route\": \"Host(`registry.skydns.local`)\"}'; \
+  etcdctl setdir /vulcand/backends/docker-registry/servers --ttl 60; \
+  while true; do \
+    export PORT=$(echo $(/usr/bin/docker port registry 5000) | cut -d':' -f2); \
+    etcdctl set /vulcand/backends/docker-registry/servers/srv1 \"{\\\"url\\\": \\\"http://$HOSTNAME:$PORT\\\"}\"; \
```
Comment (martingartonft) [inline service-files/registry-sidekick.service]: ttl?

File: service-files/registry-sidekick.service [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Description=Docker registry sidekick
+PartOf=registry.service
+After=registry.service
+
+[Service]
+ExecStart=/bin/sh -c "\
+  etcdctl set /skydns/local/skydns/registry '{"host":"127.0.0.1"}'; \
+  etcdctl set /vulcand/backends/repo-mirror/backend '{\"Type\": \"http\"}'; \
+  etcdctl set /vulcand/frontends/repo-mirror/frontend '{\"Type\": \"http\", \"BackendId\": \"docker-registry\", \"Route\": \"Host(`registry.skydns.local`)\"}'; \
+  etcdctl setdir /vulcand/backends/docker-registry/servers --ttl 60; \
```
Comment (martingartonft) [inline service-files/registry-sidekick.service]: no ttl needed here I think.

Comment (Zapadlo) [inline service-files/registry-sidekick.service]: All other sidekicks have ttl on servers, but I must admit, I don't
remember what we agreed on. Do we have a reference sidekick?

Comment (martingartonft) [inline service-files/registry-sidekick.service]: Not sure.

Maybe keep it like this for consistency then and if we don't want ttl on
servers we can change it all at once later.

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.

Comment (martingartonft) [inline service-files/registry-sidekick.service]: +1

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.

**Pull request merged**